### PR TITLE
Make 0.25.0 release

### PR DESCRIPTION
**Pull request by fitzgen**

### CODE REVIEW

**Review by pchickey — APPROVED** [review]

File: RELEASES.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## 0.25.0
 
-Not yet released
+Released 2020-03015.
```
Comment (bjorn3) [inline RELEASES.md:7]:
```suggestion
Released 2020-03-15.
```

File: RELEASES.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ## 0.25.0
 
-Not yet released
+Released 2020-03-15.
```
Comment (lqd) [inline RELEASES.md:7]:
```suggestion
Released 2021-03-15.
```

Comment (lqd) [reply RELEASES.md:7]: (we're the 16th btw, not sure if today's date was what you were going for ?)

Comment (fitzgen) [reply RELEASES.md]: Double d'oh.

**Pull request merged**